### ISSUE TITLE
[Suggestion]: Support DevToys Preview

### ISSUE THREAD

**Issue by DerStimmler** [umi-uyura/Flow.Launcher.Plugin.DevToysLauncher#6]
It would be nice if the plugin would also work with the preview version of DevToys, since that is the version that is currently linked for download on the [official website](https://devtoys.app/download).

**Comment by JesseOgunlaja** [comment]
Yeah would really love that or an option to change which directory it looks for.
